lib/src/Uncertainty/Algorithm/OrthogonalBasis/openturns/OrthogonalProductPolynomialFactory.hxx

```diff
@@ -95,6 +97,12 @@ private:
   /** Build the measure based on the one found in the family collection */
   void buildMeasure();
 
+  /** String formatter with blank space right padding */
+  String padString(const String & string, const UnsignedInteger columnWidth) const;
```

Comment:
To be moved in String.hxx as a static method

Reply:
How to do this?

Reply:
Oouuupppss! There is no String.hxx! Maybe it is time to create it...

Reply:
Is there a particular template code I may follow?

Reply:
@jschueller : I would like to implement the required static method, but I was unable to write the C++ code. May you suggest a template C++ source code from OT that I could replicate to implement that? Is there any other static method in the library that could provide the inspiration that I need?

Reply:
Fixed at 230c32b29